p2p/src/authenticated/config.rs

```diff
@@ -63,11 +63,15 @@ pub struct Config<C: Scheme> {
     /// Quota for incoming connections across all peers.
     pub allowed_incoming_connection_rate: Quota,
 
-    /// Frequency to attempt to dial known addresses.
+    /// Frequency at which we make a new dial attempt to a peer.
```

Comment:
nit: would make this clearer that this is the rate for all dialing (not just a given peer)? While it doesn't use `per peer` I find it a little opaque.